test/parser/parser-util.test.ts

```diff
@@ -1,7 +1,7 @@
 import * as path from 'path';
 import { parseDefinition } from '../../lib/parser/parser-utils';
 import { DirectiveType, ComponentRendition } from '../../lib/models';
-const cloneDeep = require('lodash.cloneDeep');
+const cloneDeep = require('lodash.clonedeep');
```

Comment:
I want 10 minutes of my life back on figuring out why the build failed in GitHub actions...

Reply:
Great debugging experience huh 😄

Reply:
The debugging experience was ok, but I was just confused why it worked locally on my Mac but not on the build server running Ubuntu...